extensions/wikia/FacebookClient/scripts/preferences.js

```diff
@@ -54,7 +58,10 @@
 				callback: function (data) {
 					if (data.status === 'ok') {
 
-						window.GlobalNotification.show($.msg('fbconnect-preferences-connected'), 'confirm');
+						bannerNotification
+							.setContent($.msg('fbconnect-preferences-connected'))
```

Comment:
@kvas-damian What do you think about creating a ticket to do so and taking care of escaping messages problem on a separate basis?